src/styles/main.scss

```
@@ -1,3 +1,3 @@
-body {
-  margin: 0;
-}
+@import "./block.scss/container.scss";
+@import "./utils.scss/normalized.scss";
+@import "./utils.scss/reset.scss"
```

2 points:
- remove `.scss` suffix from your files names
- you don't have to explicitly indicate file extension in SCSS
```suggestion
@import "./block/container";
@import "./utils/normalized";
@import "./utils/reset"
```

---

fixed